custom_functions/mark_evidence.py

```diff
@@ -49,29 +37,67 @@ def mark_evidence(container=None, object_id=None, content_type=None, **kwargs):
     else:
         raise TypeError("The input 'container' is neither a container dictionary nor an int, so it cannot be used")
     
-    # Build json dictionary for requests post
-    data = {
-        "container_id": container_id,
-        "object_id": object_id,
-        "content_type": content_type,
-        }
+    # If content added is type 'action run',
+    # then iterate through an input object that is a results object,
+    # and append the action_run_id's to data
+    if isinstance(input_object, list) and content_type == 'actionrun':
```

Comment:
Should this be content_type.lower() for consistency with the other places we check the content_type?